README.md

```diff
@@ -53,6 +53,16 @@ daemon_mode: True
 $ python3 src/cerberus.py --config <config_file_location>
 ```
 
+#### Run containerized version
+Assuming that the latest docker is intalled on the host, run:
+```
+$ cd containers
+$ docker build -t cerberus:latest .
+$ docker run --name=cerberus -v <path_to_kubeconfig>:/root/.kube/config -v <path_to_cerberus_config>:/root/cerberus/config/config.ini -d cerberus:latest
+$ docker logs -f cerberus
+```
+Similarly, podman can be used to achieve the same.
```

Comment:
Do we want to provide an example here too?

Reply:
Sure, updated the PR. PTAL.